hack/local-up-master/lib.sh

```diff
@@ -0,0 +1,302 @@
+#!/usr/bin/env bash
+
+# preserve etcd data. you also need to set ETCD_DIR.
+PRESERVE_ETCD="${PRESERVE_ETCD:-false}"
+API_PORT=${API_PORT:-8443}
+API_SECURE_PORT=${API_SECURE_PORT:-8443}
+
+# WARNING: For DNS to work on most setups you should export API_HOST as the docker0 ip address,
+API_HOST=${API_HOST:-localhost}
+API_HOST_IP=${API_HOST_IP:-"127.0.0.1"}
+ADVERTISE_ADDRESS=${ADVERTISE_ADDRESS:-""}
+FIRST_SERVICE_CLUSTER_IP=${FIRST_SERVICE_CLUSTER_IP:-10.0.0.1}
+HOSTNAME_OVERRIDE=${HOSTNAME_OVERRIDE:-"127.0.0.1"}
+CONTROLPLANE_SUDO=
+LOG_LEVEL=${LOG_LEVEL:-3}
+# Use to increase verbosity on particular files, e.g. LOG_SPEC=token_controller*=5,other_controller*=4
+LOG_SPEC=${LOG_SPEC:-""}
+WAIT_FOR_URL_API_SERVER=${WAIT_FOR_URL_API_SERVER:-60}
+MAX_TIME_FOR_URL_API_SERVER=${MAX_TIME_FOR_URL_API_SERVER:-1}
+
+#source "$(dirname "${BASH_SOURCE}")/../lib/init.sh"
+KUBE_ROOT=OS_ROOT
+source "${OS_ROOT}/hack/local-up-master/logging.sh"
+source "${OS_ROOT}/hack/local-up-master/util.sh"
+source "${OS_ROOT}/hack/local-up-master/etcd.sh"
+
+
+CONFIG=$(pwd)/openshift.local.masterup
```

Comment:
@soltysh 
> Why not using /tmp ?

I think it makes sense to set this to use /tmp when running it from test-cmd. But when running as standalone apiserver, it follows same behavior of `oc cluster up` by placing config directory in current working path